From 6de245940eea868762021c5ba7ee4c621404feeb Mon Sep 17 00:00:00 2001
From: Mladen Milinkovic <maxrd2@smoothware.net>
Date: Thu, 4 Jul 2024 01:54:10 +0200
Subject: [PATCH 05/11] VideoPlayer: remove usage of internal
 AVInputFormat.read_seek field

It's an internal field, so it should not be touched.
---
 src/videoplayer/backend/streamdemuxer.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/videoplayer/backend/streamdemuxer.cpp b/src/videoplayer/backend/streamdemuxer.cpp
index 3fb7476f..9c949fa8 100644
--- a/src/videoplayer/backend/streamdemuxer.cpp
+++ b/src/videoplayer/backend/streamdemuxer.cpp
@@ -314,10 +314,8 @@ StreamDemuxer::componentOpen(int streamIndex)
 		m_vs->audStream = ic->streams[streamIndex];
 
 		m_vs->audDec.init(avCtx, &m_vs->audPQ, nullptr, m_vs->continueReadThread);
-		if((m_vs->fmtContext->iformat->flags & (AVFMT_NOBINSEARCH | AVFMT_NOGENSEARCH | AVFMT_NO_BYTE_SEEK))
-			&& !m_vs->fmtContext->iformat->read_seek) {
+		if(m_vs->fmtContext->iformat->flags & AVFMT_NOTIMESTAMPS)
 			m_vs->audDec.startPts(m_vs->audStream->start_time, m_vs->audStream->time_base);
-		}
 		m_vs->audDec.start();
 		m_vs->audDec.pause();
 		break;
-- 
2.46.0

